model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -217,12 +217,12 @@ message ExecutableStagePayload {
   // PTransform the ExecutableStagePayload is the payload of.
   string input = 2;
 
-  // Side Input PCollection ids. Each must be present as a value in the inputs of
-  // any PTransform the ExecutableStagePayload is the payload of.
-  repeated string side_inputs = 3;
+  // The side inputs required for this executable stage. Each must be prsent as a side input of
```

Comment:
spelling

Reply:
Aren't you familiar with that common abbreviation??? It shaves off a whole character!